src/sentry/utils/snuba_rpc.py

```diff
@@ -21,6 +23,12 @@ def meta(self) -> sentry_protos.snuba.v1alpha.request_common_pb2.RequestMeta:
         ...
 
 
+class SnubaRPCException(Exception):
```

Comment:
In the other snuba file we provide a specific set of errors depending on the error code returned. Should this do something similar? https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L421